x-pack/qa/sql/src/main/resources/logs.csv

```diff
@@ -0,0 +1,101 @@
+id,@timestamp,bytes_in,bytes_out,client_ip,client_port,dest_ip,status
```

Comment:
I'd suggest to use more unique IPs for `client_ip` and `destip` so that is clearer that order by works correctly. Currently all IPs here: https://github.com/elastic/elasticsearch/pull/34758/files#diff-f9d17e274b4800d6b8f0fa433d1bce17R44 are the same.

Reply:
And some missing IPs in the data? To show null handling.

Reply:
I've put the file through excel randomizer plus removed some IPs.